### PR TITLE
Create builders used for test cases 

### LINKED ISSUE #84: Object builders for testing

### PR DESCRIPTION
**Pull request by Ruofan1023**
Also, write some sample test cases for delete module to test if the builders work :/
Fixes #84

### CODE REVIEW

File: src/test/java/seedu/address/testutil/StudentBuilder.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+     * Sets the {@code studentId} of the {@code Student} that we are building.
+     */
+    public StudentBuilder withTasks(List<Task> taskList) {
+        this.taskList = taskList;
+        return this;
+    }
+
+    /**
+     * Build a typical student
+     * @return
```
Comment (felissaf) [inline src/test/java/seedu/address/testutil/StudentBuilder.java:84]: returns typical student?

File: src/test/java/seedu/address/testutil/ModuleBuilder.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+     * Sets the {@code tasks} of the {@code Module} that we are building.
+     */
+    public ModuleBuilder withTasks(List<Task> tasks) {
+        this.tasks = tasks;
+        return this;
+    }
+
+    /**
+     * Build a typical module.
+     * @return
```
Comment (felissaf) [inline src/test/java/seedu/address/testutil/ModuleBuilder.java:62]: returns typical module?

File: src/main/java/seedu/address/model/task/Task.java [hunk 3]
```diff
@@ -36,6 +36,10 @@ public String getModule() {
         return this.module.toString();
     }
 
+    public ModuleName getModuleName() {
+        return this.module;
+    }
```
Comment (tanjoewel) [inline src/main/java/seedu/address/model/task/Task.java:41]: Maybe I can suggest changing the ModuleName field in task from `module` to `moduleName`?

File: src/test/java/seedu/address/testutil/TypicalStudents.java [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    public static final Student GEORGE = new StudentBuilder()
+            .withStudentId("A1234566A")
+            .withName("George Best")
+            .withEmail("anna@example.com")
+            .withTeleHandle("@George")
+            .build();
+
+    public static List<Student> getTypicalStudents() {
+        return new ArrayList<>(Arrays.asList(ALICE, BENSON, CARL, DANIEL, ELLE, FIONA, GEORGE));
+    }
```
Comment (tanjoewel) [inline src/test/java/seedu/address/testutil/TypicalStudents.java:57]: Should there be javadocs for public method?

**Review by tanjoewel — COMMENTED** [review]
Looks mostly good :>

File: src/test/java/seedu/address/testutil/StudentBuilder.java (hunk 1 above)
Comment (Ruofan1023) [reply src/test/java/seedu/address/testutil/StudentBuilder.java:84]: Oh yes! forget to add

File: src/main/java/seedu/address/model/task/Task.java (hunk 3 above)
Comment (Ruofan1023) [reply src/main/java/seedu/address/model/task/Task.java:41]: yep that's better

**Review by charliemoweng — COMMENTED** [review]
LGTM

**Pull request merged**